bee-network/src/network/host.rs

```diff
@@ -51,7 +52,9 @@ pub mod integrated {
 
         async fn start(node: &mut N, config: Self::Config) -> Result<Self, Self::Error> {
             node.spawn::<Self, _, _>(|shutdown| async move {
-                network_host_processor(config, shutdown).await;
+                network_host_processor(config, shutdown)
+                    .await
+                    .expect("network host processor.");
```

Comment:
Perhaps a more descriptive message here.

Reply:
Are you sure that's necessary. The error is printed alongside this message. Thibault thinks, this additional message isn't necessary at all. I am not fully decided, but I think it can provide additional context in debugging scenarios to help reasoning in the terminal already without having to look up the code line where the error was thrown.